cypress/e2e/pseudonimized.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { FolderItemFactory, ItemLoginSchemaType } from '@graasp/sdk';
+
+import { buildContentPagePath } from '@/config/paths';
+import {
+  ENROLL_BUTTON_SELECTOR,
+  ITEM_LOGIN_SIGN_IN_BUTTON_ID,
+  ITEM_LOGIN_USERNAME_INPUT_ID,
+  buildDataCyWrapper,
+} from '@/config/selectors';
+import { ID_FORMAT } from '@/utils/item';
+
+import { API_HOST } from '../support/env';
+import { DEFAULT_POST } from '../support/utils';
+
+describe('Pseudonimized access', () => {
+  it('Logged out', () => {
+    cy.intercept({
+      method: DEFAULT_POST.method,
+      url: new RegExp(`${API_HOST}/items/${ID_FORMAT}/login$`),
+    }).as('itemLoginSignIn');
+
+    const rootItem = FolderItemFactory({ name: 'pseudo' });
+    const items = [rootItem];
+    cy.setUpApi({
+      currentMember: null,
+      items,
+      itemLogins: { [rootItem.id]: `${ItemLoginSchemaType.Username}` },
+    });
```

Comment:
Usually we do this at the start of the test, should we move it there to be consistent ?